### DISCUSSION TITLE
Is there a plan to implement the feature to execute cell range commands?

### DISCUSSION THREAD

**Discussion by baiguoname** [evcxr/evcxr#382] [General]
For example in python jupyter, I can do this:
![image](https://github.com/user-attachments/assets/bfda8956-26ab-4aad-a3a1-81fd5568b719)
The cell range comand `%%time` evaluates the time consumming taken to executing the entire cell.
If I can do this in evcxr, I could run this:
![image](https://github.com/user-attachments/assets/46effccc-5c3f-485d-a473-e1799a1cc5f8)
So I even don't have to create a new `evcxr.toml` file. This is very convient.

**Comment by davidlattimore** [comment]
Currently you can put commands at the start of a cell. If I'm understanding what you're asking correctly, is this a way to have commands that only affect that cell's execution and don't make permanent changes?

**Reply by baiguoname** [reply]
> Currently you can put commands at the start of a cell. If I'm understanding what you're asking correctly, is this a way to have commands that only affect that cell's execution and don't make permanent changes?

However, the current available commands don't support line break. An cell range command affects the entire cell, it can make permanet changes.

**Comment by davidlattimore** [comment]
Ah, gotcha, now I understand. That seems like a reasonable feature to add. I don't have any plans to add it myself at this stage, but I'm happy to review PRs for it

**Reply by baiguoname** [reply]
Wonderful, I'm glad to make a pr to implement this feature, I'm going to try to implement this feature shortly.
